From 2b03cce772295062439d07e0fa128a5ab7e1c48b Mon Sep 17 00:00:00 2001
From: Sridhar Gujje <sgujje@codeaurora.org>
Date: Thu, 12 Sep 2013 18:04:08 +0530
Subject: [PATCH 8/8] core: squashed commit of:

Initial bringup system-core

Qcril : Added diag permission to Ril

diag: Remove world read/write permissions from /dev/diag

/dev/diag has world read/write permissions which is considered
a security risk.  Remove these permissions.  Add new qcom_diag
group for access to /dev/diag

CRs-Fixed: 356415

Change-Id: I41c950d9b24742b25c4eada63d62d493550c384d
Conflicts:
	include/private/android_filesystem_config.h
	rootdir/init.rc
---
 include/system/graphics.h | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/include/system/graphics.h b/include/system/graphics.h
index 5b76c50..8a70153 100644
--- a/include/system/graphics.h
+++ b/include/system/graphics.h
@@ -54,7 +54,8 @@ enum {
     HAL_PIXEL_FORMAT_RGB_888            = 3,
     HAL_PIXEL_FORMAT_RGB_565            = 4,
     HAL_PIXEL_FORMAT_BGRA_8888          = 5,
-
+    HAL_PIXEL_FORMAT_RGBA_5551          = 6,
+    HAL_PIXEL_FORMAT_RGBA_4444          = 7,
     /*
      * sRGB color pixel formats:
      *
-- 
1.9.1

